readme.md

```diff
@@ -584,6 +584,7 @@
 	- [slonik](https://github.com/gajus/slonik) - PostgreSQL client with strict types, detailed logging and assertions.
 	- [Objection.js](https://github.com/Vincit/objection.js) - Lightweight ORM built on the SQL query builder Knex.
 	- [TypeORM](https://github.com/typeorm/typeorm) - ORM for PostgreSQL, MariaDB, MySQL, SQLite, and more.
+	- [Endb](https://github.com/endb/endb) - Easy-to-use & simplistic ORM for SQLite3 with user-friendly API.
```

Comment:
It's not just for SQLite3.

Reply:
This is outdated pull-request, Endb has improved significantly since then. Thus, the description is irrelevant at this point. I am really sorry for not mentioning this earlier.